dev/ci/tools/runner-maintanance/CiRunnerMaintenance_ConfigureCygwinProfile.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Isn't this file the same as the dev/build/windows/configure_profile.sh?

Reply:
Yes, the other file is also derived from there. But it serves a completely different purpose and I didn't think it makes a lot of sense to link them together. I am open to better ideas of handling this.

Reply:
P.S.: the goal of this is to collect scripts and documentation to

- completely finish the setup of a runner from a fresh Windows VM
- efficiently maintain the operation of the runners

If someone has scripts and documentation to install and configure the gitlab runner elsewhere, this should go together.